packages/ember-glimmer/lib/renderer.js

```diff
@@ -223,7 +223,7 @@ class Renderer {
   }
 
   _unregister(view) {
-    delete this._viewRegistry[this.elementId];
+    delete this._viewRegistry[view.elementId];
```

Comment:
Seems fine to change here, but this file isn't used at all on 2.8/2.9 (since the ember-glimmer flag is off).